fvp.sh

```diff
@@ -36,10 +36,35 @@ if ! docker image inspect "fvp:${FVP_VERSION}" >/dev/null 2>&1; then
     "${DIRNAME}/build.sh"
 fi
 
+
+# Set mount_dir from environment variable or default to pwd
+mount_dir="${FVP_MOUNT_DIR:-$(pwd)}"
+
+# Validate mount_dir exists and is a directory
+if [[ ! -d "$mount_dir" ]]; then
+    echo "Error: FVP_MOUNT_DIR '$mount_dir' is not a valid directory" >&2
+    exit 1
+fi
+
+# Set workdir from environment variable or default to mount_dir
+workdir="${FVP_WORKDIR:-$mount_dir}"
```

Comment:
Can we keep the default as `pwd`?

Reply:
https://github.com/Arm-Examples/FVPs-on-Mac/pull/4/commits/6df5e9c06c5ec9cf89090d0cb0ddaf6120e98ba8